client/src/components/CardOffering.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import Image from "../components/Image";
+
+import Container from "./Container";
+import Toggle from "./IconToggle";
+import snowboard from "../assets/images/snowboard.jpg";
+
+export default function CardOffering() {
+  return (
+    <CardContainer>
+      <ToggleRight />
+      <H4>snowboard</H4>
+
+      <Img src={snowboard} />
+    </CardContainer>
+  );
+}
+
+const ToggleRight = styled(Toggle)`
+  margin: 0;
+`;
+
+const Img = styled(Image)`
+  width: 100%;
+  margin: 0 auto;
```

Comment:
Maybe you find a way to definde the positions of your elements without margin, instead let the outer element define the size and position :) (Just my personal opinion and of course, for now i dont see the rendered version)